cmake/modules/RootBuildOptions.cmake

```diff
@@ -117,6 +117,7 @@ ROOT_BUILD_OPTION(coverage OFF "Enable compile flags for coverage testing")
 ROOT_BUILD_OPTION(cuda OFF "Enable support for CUDA (requires CUDA toolkit >= 7.5)")
 ROOT_BUILD_OPTION(cudnn ON "Enable support for cuDNN (default when Cuda is enabled)")
 ROOT_BUILD_OPTION(cxxmodules OFF "Enable support for C++ modules")
+ROOT_BUILD_OPTION(daos OFF "Enable support for Intel DAOS")
```

Comment:
Should we rephrase it as "Enable support for Intel DAOS for RNTuple"?

Reply:
Or "Enable RNTuple support for Intel DAOS"?

Reply:
Changed.